pages/team.vue

```diff
@@ -170,8 +170,14 @@ export default {
     ...mapGetters('team', ['members'])
   },
   created() {
-    this.setTeamRef()
+    this.setCoreTeamRef()
+    this.setFacultyTeamRef()
   },
+  mounted() {
```

Comment:
We don't need mounted here. Once the `reference` has been set to firestore using `bindFirestoreRef`, it listens to realtime updates in `DB`. Hence, we don't need to set it again and again